databases/week2/part 4.sql

```diff
@@ -0,0 +1,22 @@
+create table customer(
+id int primary key,
+name varchar(255),
+city varchar(255),
+zip int,
+street varchar(255) not null,
+mobile int not null,
+email varchar(255) not null);
+
+create table orders(
+id int primary key,
+total_price int );
+
+create table order_lines(
+orders_id int primary key,
```

Comment:
This table also need a primary key, number_of_products and also a price and a total.
If you want to be more precise you also need a product_prices, but the prices are copied to the order_lines.
I hope that I am not very demanding